airgun/views/template.py

```diff
@@ -0,0 +1,42 @@
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+
+from airgun.views.common import BaseLoggedInView
+from airgun.widgets import ActionsDropdown
+from airgun.widgets import RadioGroup
+
+
+class SyncTemplatesView(BaseLoggedInView):
+    title = Text("//h2[contains(., 'Import or Export Templates')]")
+    import_export = RadioGroup("//div[@class='col-md-6']")
+    submit = Text("//button[@type='submit']")
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(self.title, exception=False) and \
+               self.browser.wait_for_element(self.import_export) is not None
+
+    @View.nested
+    class ImportTemplates(View):
```

Comment:
As mentioned, these should use `ConditionalSwitchableView`. You can find many examples in Airgun, but it would go something like this:
```python
class SyncTemplatesView(BaseLoggedInView):
    import_export = RadioGroup("//div[@class='col-md-6']")
    action_type = ConditionalSwitchableView(reference='import_export')

    @action_type.register('Import')
    class ImportTemplates(View):
        associate = ActionsDropdown("//select[@name='associate']")
        # Others skipped for brevity

    @action_type.register('Export')
    class ExportTemplates(View):
        branch = TextInput("//input[@name='branch']")
        # Others skipped for brevity
```

Written like that, on robottelo you will need to do something like:
```
session.sync_template.import({'import_export': 'Import', 'associate': 'some value'})
```
Since `import()` action will always select `Import`, and `export()` will always select `Export`, it makes sense to relieve user from specifying that option manually. Just add this key with specific value to `values` in `SyncTemplatesEntity`.

Or re-write this entity to expose single action, say `submit`, and require user to specify this key:
```
session.sync_template.sync({'import_export': 'Import', 'associate': 'some value'})
```